### PR TITLE
layerscape: bump layerscape packages to lf-6.6 and add 6.6 kernel as testing

### PR DESCRIPTION
**Pull request by CHKDSK88**
This PR introduces 6.6 testing kernel for layerscape and bumps target-specified packages to lf-6.6 vversion.

@aparcar 
Please look if packages are ready for apk now.

### CODE REVIEW

**Comment by namiltd** [comment]
missing: # VIDEO_IMX8MQ_MIPI_CSI2 is not set

**Comment by CHKDSK88** [comment]
> missing: # VIDEO_IMX8MQ_MIPI_CSI2 is not set

Yeah, and CONFIG_VIDEO_IMX8_ISI too. I added both.

File: package/boot/tfa-layerscape/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tfa-layerscape
-PKG_VERSION:=lf-6.1.1-1.0.0
-PKG_RELEASE:=1
+PKG_VERSION:=lf_6.6.3_1.0.0
+PKG_RELEASE:=r1
```
Comment (aparcar) [inline package/boot/tfa-layerscape/Makefile:12]: The r is now automatically added via the build system, no need to define it

File: package/boot/tfa-layerscape/Makefile [hunk 2]
```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tfa-layerscape
-PKG_VERSION:=lf-6.1.1-1.0.0
-PKG_RELEASE:=1
+PKG_VERSION:=lf_6.6.3_1.0.0
```
Comment (aparcar) [inline package/boot/tfa-layerscape/Makefile:11]: A version can no longer contain a string. Please change this to 6.6.3.1.0.0

File: package/boot/tfa-layerscape/Makefile (hunk 1 above)
Comment (CHKDSK88) [reply package/boot/tfa-layerscape/Makefile]: I fixed all bumps.

**Comment by hauke** [comment]
Please rebase this on top of master.

File: target/linux/layerscape/image/armv8_64b.mk [hunk 3]
```diff
@@ -157,7 +157,6 @@ define Device/fsl_ls1043a-rdb
     kmod-ahci-qoriq \
     kmod-hwmon-ina2xx \
     kmod-hwmon-lm90
-  DEVICE_DTS := freescale/fsl-ls1043a-rdb
```
Comment (hauke) [inline target/linux/layerscape/image/armv8_64b.mk:160]: Why do you remove this entry here and do not add it again?

File: target/linux/layerscape/image/armv8_64b.mk [hunk 4]
```diff
@@ -203,7 +202,6 @@ define Device/fsl_ls1046a-frwy
   DEVICE_PACKAGES += \
     layerscape-fman \
     trusted-firmware-a-ls1046a-frwy
-  DEVICE_DTS := freescale/fsl-ls1046a-frwy
```
Comment (hauke) [inline target/linux/layerscape/image/armv8_64b.mk:206]: Why is this only removed?

File: target/linux/layerscape/image/armv8_64b.mk [hunk 5]
```diff
@@ -249,7 +247,6 @@ define Device/fsl_ls1046a-rdb
     kmod-ahci-qoriq \
     kmod-hwmon-ina2xx \
     kmod-hwmon-lm90
-  DEVICE_DTS := freescale/fsl-ls1046a-rdb
```
Comment (hauke) [inline target/linux/layerscape/image/armv8_64b.mk:252]: Why is this only removed?

File: target/linux/layerscape/image/armv8_64b.mk (hunk 3 above)
Comment (hauke) [reply target/linux/layerscape/image/armv8_64b.mk:160]: Sorry, the default is working: `  DEVICE_DTS = $(subst _,-,$(1))`.

**Comment by hauke** [comment]
Is kernel 6.6 working fine?

If it is working fine please create a PR changing the default kernel to 6.6 and removing kernel 6.1.

**Comment by CHKDSK88** [comment]
> Is kernel 6.6 working fine?
> 
> If it is working fine please create a PR changing the default kernel to 6.6 and removing kernel 6.1.

@hauke
I will prepare PR next week.

**Pull request merged**